src/qt/overviewpage.cpp

```diff
@@ -321,12 +321,10 @@ void OverviewPage::showOutOfSyncWarning(bool fShow)
 
 void OverviewPage::updatePrivateSendProgress()
 {
-    if(!clientModel->masternode().isBlockchainSynced() || ShutdownRequested()) return;
-
-    if(!walletModel) return;
+    if(ShutdownRequested() || !walletModel || !clientModel || !clientModel->masternode().isBlockchainSynced()) return;
```

Comment:
Since you're already changing this and it's dash specific
```suggestion
    if (ShutdownRequested() || !walletModel || !clientModel || !clientModel->masternode().isBlockchainSynced()) return;
```